tests/compat-bpf.sh

```diff
@@ -44,34 +44,11 @@ begin_test "rule should already exist"
 )
 end_test
 
-begin_test "validate nft output: hook"
+begin_test "json diff"
 (
-    $NFT_LIST_TABLE ip $TABLE
-    $NFT_LIST_TABLE_JSON ip $TABLE | $JQ .nftables[].chain.hook | grep input
+    $NFT_LIST_TABLE_JSON ip $TABLE | python3 $BASEDIR/tests/py/compare.py $BASEDIR/tests/fixtures/ip-fixture-filter.json
```

Comment:
Please rename the fixture files to something like `compat-bpf-pinned.json`, `compat-bpf-filter.json` and `compat-sets.json`